### ISSUE TITLE
Back arrow of settings sub menu returns to main screen

### ISSUE THREAD

**Issue by lindoman** [mpcjanssen/simpletask-android#1175]
This is a usability enhancement request. When I click on the back arrow at the top of a settings sub menu, it returns me two steps to the main application screen, not the settings menu screen I just came from. If I press my android back button, it behaves the way I would have expected the arrow at the top of the screen to work (it returns me to the settings menu screen). Half the time I remember, and while I am still learning where all the settings are, it is quite frustrating! I would love this to be fixed if possible.

**Comment by smichel17** [comment]
Which version of Android and SimpleTask? The back arrow behaves the way you would expect for me (ST v10.9.3 on Android 7.0).

**Comment by lindoman** [comment]
Just for clarity, I am talking about the "in app" back arrow at the top of the screen, not the android back arrow (which still works as expected). I'm on version 11.0.0 on android 12, but this was an issue on the previous version I had installed. I doubt know if this helps, but the screen does seem to "redraw" itself somehow (very brief "darkness" to the screen before showing the main todo list)

**Comment by smichel17** [comment]
Yep same arrow. I would guess it's (yet another) change on Android that broke something which previously worked fine.

**Comment by lindoman** [comment]
Ahh the joy of being an android developer... Sounds like that happens more often than you would like! Thanks for the reply, I assume it's ok to leave this issue open anyway? Let me know if you ever want a fix to be tested.

**Comment by smichel17** [comment]
> I assume it's ok to leave this issue open anyway?

Definitely. Next step is to see if it's reproducible in an Android 12 emulator

**Comment by mpcjanssen** [comment]
I ff-ing (excusé le mot) hate it that basic stuff like this is so hard. It's OK to leave it open but don't expect a fix soon.

**Comment by lindoman** [comment]
No worries, there is a workaround anyway by using the android button. Thank you for the software, and your "mot" is excused (and justified!)
